signac/contrib/project.py

```diff
@@ -790,7 +791,7 @@ def _contains_job_id(self, job_id):
 
         """
         # We can rely on the project workspace to be well-formed, so just use
-        # str.join with os.sep instead of os.path.join for speed.
+        # os.sep.join instead of os.path.join for speed.
         return os.path.exists(os.sep.join((self.workspace(), job_id)))
```

Comment:
Will `self.workspace()` become a property instead of a method? How are we handling that?

Reply:
Good question. Yes, it should become a property.